spec/API_specification/array_object.md

```diff
@@ -1249,6 +1249,12 @@ Element-wise results must equal the results returned by the equivalent element-w
 
 Evaluates `self_i / other_i` for each element of an array instance with the respective element of the array `other`.
 
+```{note}
+If one or both of `self` and `other` have integer data types, the result is implementation-dependent. This is because type promotion between data type “kinds” (i.e. integer versus floating-point) is unspecified.
+
+Specification-compliant libraries may choose to raise an error or return an array containing the element-wise results. If an array is returned, it must have a floating-point data type.
```

Comment:
Here the distinction of "result" is the actual values returned which is separate from the requirement that the output dtype must be floating-point. What this note is getting at is that, because implementations may vary in how they promote between different "kinds", what values are returned may also vary, but the output result dtype must be floating-point regardless.

Reply:
I'll mark this conversation as resolved.